project_event/tests/test_project_event_task.py

```diff
@@ -459,7 +472,96 @@ def test_190_main_task_activity_calendar_event_clone_state(self):
             calendar_event = child.get_calendar_event()
             self.assertEqual(calendar_event.state, 'draft')
 
-    def test_200_get_parent_project_id(self):
+    def test_200_compute_actual_total_time(self):
+        # Test case 1: Testing for same actual start and end date
+        date_start_1 = datetime.today()
+        date_end_1 = date_start_1 + timedelta(hours=2)
+        actual_date_start_1 = datetime.today()
+        actual_date_end_1 = actual_date_start_1
+        task_vals_1 = {
+            'name': 'Sample Task 1',
+            'activity_task_type': 'task',
+            'partner_id': self.project_1.partner_id.id,
+            'room_id': self.room_1.id,
+            'date_start': fields.Datetime.to_string(date_start_1),
+            'date_end': fields.Datetime.to_string(date_end_1),
+            'real_date_start': fields.Datetime.to_string(actual_date_start_1),
+            'real_date_end': fields.Datetime.to_string(actual_date_end_1),
```

Comment:
You can simply: 'real_date_end': fields.Datetime.to_string(actual_date_start_1),

Reply:
@rimbendhaou : That is for more clarity and readability. If the test case changes, we can change the value of the variable. if variables are reading from config files in the future, it makes sense.